pkg/test/framework/analyzer.go

```diff
@@ -99,11 +99,15 @@ func (s *suiteAnalyzer) RequireSingleCluster() Suite {
 	return s.RequireMinClusters(1).RequireMaxClusters(1)
 }
 
-func (s *suiteAnalyzer) RequireEnvironmentVersion(version string) Suite {
+func (s *suiteAnalyzer) RequireMinVersion(version string) Suite {
```

Comment:
nit: since we have `RequireIstioVersion` now should we rename to remove ambiguity?

Reply:
maybe `RequireMinKubernetesVersion`? not sure